bench/Benchmark.hs

```diff
@@ -60,13 +60,7 @@ circlerProgram =
     let forever : cmd () -> cmd () = \c. c; forever c
     in forever (
       move;
-      turn east;
-      move;
-      turn south;
-      move;
-      turn west;
-      move;
-      turn north
+      turn right;
```

Comment:
Let's change this to `turn right; move; turn right; move; ...` i.e. with the same number of turns and moves as before.  Otherwise the benchmark won't really be comparable to before, since the number of straight-line instructions the robot executes vs the number of recursive calls to `forever` can definitely make a performance difference.

Reply:
Indeed it does seem to be 20% slower, I should have checked :+1:

Reply:
Fixed :heavy_check_mark: